js/main.js

```diff
@@ -1,12 +1,24 @@
-import { renderCalendarCells, renderWeekHeader } from './calendar.js';
-import { initModalEventListeners } from './modal.js';
-import { addEventListenersToHeaderButtons} from './header.js';
-const currentDate = new Date();
-function init() {
-    renderCalendarCells();
-    initModalEventListeners();
-    renderWeekHeader(currentDate);
-    addEventListenersToHeaderButtons();
-}
-
-init();
\ No newline at end of file
+import { renderHeader } from './header.js';
+import { renderAside } from './aside.js';
+import { renderCalendarCells } from './calendar.js';
+import { initModal } from './modal.js';
+
+const init = () => {
+  let currentDate = new Date();
+  const { openModal } = initModal();
+
+  const updateDate = (newDate) => {
+    if (newDate.toDateString() === currentDate.toDateString()) {
```

Comment:
I don't think toDateString is correct way to compare if dates are same, since it is more meant for date representation in a string format.

Since I guess you want to compare just year, month and day, you should go with getFullYear, getMonth and getDate methods.